src/main/java/htsjdk/tribble/readers/TabixReader.java

```diff
@@ -111,6 +114,17 @@ public TabixReader(final String fn, final String idxFn) throws IOException {
     }
 
     /**
+     * @param fn File name of the data file
+     * @param idxFn Full path to the index file. Auto-generated if null
```

Comment:
`Fn` is too similar to `Function` (which we also have here, now) -- recommend renaming to `indexPath` (here and below)